kalite/distributed/static/js/distributed/content/models.js

```diff
@@ -77,17 +77,23 @@ window.ContentLogModel = ExtraFieldsBaseModel.extend({
         "content_source",
         "content_kind",
         "progress",
-        "views"
+        "views",
+        "latest_activity_timestamp"
     ],
 
+    initialize: function() {
+        _.bindAll(this);
+    },
+
     urlRoot: function() {
         return window.sessionModel.get("GET_CONTENT_LOGS_URL");
     },
 
-    save: _.throttle(function(){this.saveNow();}, 30000),
+    save: _.throttle(function(key, val, options){this.saveNow(key, val, options);}, 30000),
```

Comment:
I realize there's no significant change here, but why is this function throttled like this? A n explanatory comment would be helpful.